migration/FromMySqlToPostgreSql/MapDataTypes.php

```diff
@@ -164,7 +164,7 @@ private function __construct()
 
         'point' => [
             'increased_size'           => '',
```

Comment:
Error output I see is: 
        --Following MySQL query will return a data set, rejected by PostgreSQL:
SELECT `ID`,HEX(AsWKB(`GEOPOINT`)),`LATITUDE`,`LONGITUDE` FROM `testGeoPoint` LIMIT 0, 1;

        -- FromMySqlToPostgreSql::populateTableWorker

        -- PDOException code: 22P02
        -- File: /home/mores/wip/MySQL2PSQL/php/migration/FromMySqlToPostgreSql/FromMySqlToPostgreSql.php
        -- Line: 674
        -- Message: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for type point: "0101000000022B8716D98252C09C33A2B437584440"
CONTEXT:  COPY testGeoPoint, line 1, column GEOPOINT: "0101000000022B8716D98252C09C33A2B437584440"
        -- SQL: SELECT `ID`,HEX(AsWKB(`GEOPOINT`)),`LATITUDE`,`LONGITUDE` FROM `testGeoPoint` LIMIT 0, 1;
COPY "test"."testGeoPoint" FROM '/home/mores/wip/MySQL2PSQL/php/temporary_directory/testGeoPoint0.csv' DELIMITER ',' CSV;

        -------------------------------------------------------